Installation issue: bolt

Running:

`spack install bolt`

Fails like:

```
==> Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

7 errors found in build log:
     175    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E make_directory /tmp/root/s
            pack-stage/spack-stage-NfiX8N/spack-build/external/libabt-prefix/tm
            p
     176    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E make_directory /tmp/root/s
            pack-stage/spack-stage-NfiX8N/spack-build/external/libabt-prefix/sr
            c/libabt-stamp
     177    Can't locate Data/Dumper.pm in @INC (@INC contains: /tmp/root/spack
            -stage/spack-stage-NfiX8N/spack-src/runtime/tools/lib /usr/local/li
            b64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/
            share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at /tm
            p/root/spack-stage/spack-stage-NfiX8N/spack-src/runtime/tools/lib/t
            ools.pm line 81.
     178    BEGIN failed--compilation aborted at /tmp/root/spack-stage/spack-st
            age-NfiX8N/spack-src/runtime/tools/lib/tools.pm line 81.
     179    Compilation failed in require at /spack/var/spack/stage/bolt-1.0b1-
            jenaxkneyprxgq6abwaihlkuuoko4pwv/spack-src/runtime/tools/message-co
            nverter.pl line 23.
     180    BEGIN failed--compilation aborted at /spack/var/spack/stage/bolt-1.
            0b1-jenaxkneyprxgq6abwaihlkuuoko4pwv/spack-src/runtime/tools/messag
            e-converter.pl line 23.
  >> 181    make[2]: *** [runtime/src/kmp_i18n_default.inc] Error 2
     182    make[2]: *** Waiting for unfinished jobs....
     183    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E make_directory /tmp/root/s
            pack-stage/spack-stage-NfiX8N/spack-build/external/libabt-prefix/sr
            c
     184    Can't locate Data/Dumper.pm in @INC (@INC contains: /tmp/root/spack
            -stage/spack-stage-NfiX8N/spack-src/runtime/tools/lib /usr/local/li
            b64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/
            share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at /tm
            p/root/spack-stage/spack-stage-NfiX8N/spack-src/runtime/tools/lib/t
            ools.pm line 81.
     185    BEGIN failed--compilation aborted at /tmp/root/spack-stage/spack-st
            age-NfiX8N/spack-src/runtime/tools/lib/tools.pm line 81.
     186    Compilation failed in require at /spack/var/spack/stage/bolt-1.0b1-
            jenaxkneyprxgq6abwaihlkuuoko4pwv/spack-src/runtime/tools/message-co
            nverter.pl line 23.
     187    BEGIN failed--compilation aborted at /spack/var/spack/stage/bolt-1.
            0b1-jenaxkneyprxgq6abwaihlkuuoko4pwv/spack-src/runtime/tools/messag
            e-converter.pl line 23.
  >> 188    make[2]: *** [runtime/src/kmp_i18n_id.inc] Error 2
     189    make[2]: Leaving directory `/tmp/root/spack-stage/spack-stage-NfiX8
            N/spack-build'
  >> 190    make[1]: *** [runtime/src/CMakeFiles/libomp-needed-headers.dir/all]
             Error 2
     191    make[1]: *** Waiting for unfinished jobs....
     192    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E make_directory /tmp/root/s
            pack-stage/spack-stage-NfiX8N/spack-build/external/libabt-prefix/sr
            c/libabt-stamp
     193    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E touch /tmp/root/spack-stag
            e/spack-stage-NfiX8N/spack-build/external/libabt-prefix/src/libabt-
            stamp/libabt-mkdir
     194    [ 11%] No download step for 'libabt'
     195    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E echo_append
     196    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E touch /tmp/root/spack-stag
            e/spack-stage-NfiX8N/spack-build/external/libabt-prefix/src/libabt-
            stamp/libabt-download

     ...

     199    [ 16%] No update step for 'libabt'
     200    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E echo_append
     201    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E touch /tmp/root/spack-stag
            e/spack-stage-NfiX8N/spack-build/external/libabt-prefix/src/libabt-
            stamp/libabt-patch
     202    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external &&
             /spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cmake-3.14.5-6semn
            25gnvvlrj53qk2kxjccdqa5kav2/bin/cmake -E touch /tmp/root/spack-stag
            e/spack-stage-NfiX8N/spack-build/external/libabt-prefix/src/libabt-
            stamp/libabt-update
     203    [ 19%] Performing configure step for 'libabt'
     204    cd /tmp/root/spack-stage/spack-stage-NfiX8N/spack-build/external/li
            babt-prefix/src/libabt-build && /spack/var/spack/stage/bolt-1.0b1-j
            enaxkneyprxgq6abwaihlkuuoko4pwv/spack-src/external/argobots/configu
            re --prefix=/spack/opt/spack/linux-centos7-x86_64/gcc-7.3.0/bolt-1.
            0b1-jenaxkneyprxgq6abwaihlkuuoko4pwv CC=/spack/lib/spack/env/gcc/gc
            c
  >> 205    /bin/sh: /spack/var/spack/stage/bolt-1.0b1-jenaxkneyprxgq6abwaihlku
            uoko4pwv/spack-src/external/argobots/configure: No such file or dir
            ectory
  >> 206    make[2]: *** [external/libabt-prefix/src/libabt-stamp/libabt-config
            ure] Error 127
     207    make[2]: Leaving directory `/tmp/root/spack-stage/spack-stage-NfiX8
            N/spack-build'
  >> 208    make[1]: *** [external/CMakeFiles/libabt.dir/all] Error 2
     209    make[1]: Leaving directory `/tmp/root/spack-stage/spack-stage-NfiX8
            N/spack-build'
  >> 210    make: *** [all] Error 2

See build log for details:
  /spack/var/spack/stage/bolt-1.0b1-jenaxkneyprxgq6abwaihlkuuoko4pwv/spack-build-out.txt
```

--- a/var/spack/repos/builtin/packages/bolt/package.py
+++ b/var/spack/repos/builtin/packages/bolt/package.py
@@ -22,6 +22,10 @@ class Bolt(CMakePackage):
 
     version("1.0b1", "df76beb3a7f13ae2dcaf9ab099eea87b")
 
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
     def cmake_args(self):
         options = [
             '-DLIBOMP_USE_ITT_NOTIFY=off',

